Uchu.StandardScripts/Equipment/SkillSetTriggerTemplate.cs

```diff
@@ -0,0 +1,65 @@
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.Equipment
+{
+    /// <summary>
+    /// Native implementation of scripts/equipmenttriggers/skillsettriggertemplate.lua
+    /// </summary>
+    //[ScriptName("skillsettriggertemplate.lua")]
+    public class SkillSetTriggerTemplate : ObjectScript
```

Comment:
A lot of this class is commented out. Is there a reason?

Reply:
A lot of old variables that I forgot to remove completely